AmazonChimeSDKDemo/AmazonChimeSDKDemo/MeetingModel.swift

```diff
@@ -526,7 +516,14 @@ extension MeetingModel: VideoTileObserver {
                 videoModel.addRemoteVideoTileState(tileState, completion: { success in
                     if success {
                         if self.activeMode == .video {
+                            // If the video is not currently being displayed, pause it
+                            if !self.videoModel.remoteVideoStatesInCurrentPage.contains(where: { $0.0 == tileState.tileId }) {
```

Comment:
Can we extract this into a method on VideoModel? `remoteVideoStatesInCurrentPage` can also become private afterwards.
`func isDisplaying(tileId: Int) -> Bool`

Reply:
Updated.